docs/deploy-rbd.md

```diff
@@ -299,7 +299,7 @@ it in a Kubernetes Secret (by default) called `ceph-csi-kms-token`, where the
 Vault Token is stored in the `token` key as shown in [the
 example](../examples/kms/vault/tenant-token.yaml).
 
-#### Configuring HashiCorp Vault with Kubernetes ServiceAccount
+#### Configuring HashiCorp Vault with a single Kubernetes ServiceAccount
```

Comment:
nit: s/single/Single/ ?

Reply:
"Single" is not a name of a component/project/object, or start of the title, so I kept it lowecase

Reply:
This is a title case, so it should be **Single** :-)

Reply:
Ah! please ignore, I see now that nowhere the doc is following the title case.